web/viewer.css

```diff
@@ -1419,6 +1419,7 @@ html[dir='rtl'] .outlineItemToggler::before {
 .dialog .buttonRow {
   text-align: center;
   vertical-align: middle;
+  clear: both;
```

Comment:
Previously, `#mozPrintCallback-shim .progress-actions` required `clear: both;`. I wanted to be able to simply reuse this `.buttonRow` class, so it seemed straightforward enough for me to just lump it in here. It has no visible implications on anything else using this class, though if we're concerned about style property bloat I could make a new selector with more specificity.